sail/app/gora/voyer.hoon

```diff
@@ -71,16 +71,52 @@
       %ignore-give
     ?.  (~(has by args) 'gor')  'ゴラが必要'
     [%ignore-give (slav %uv (~(got by args) 'gor'))]
+  ::
+      %create-gora-qr
+    ?.  (~(has by args) 'expiration')  '有効期限を時間単位で入力してください'
+    ?.  (~(has by args) 'gor')  'ゴラが必要'
+    =/  gor-id  (slav %uv (~(got by args) 'gor'))
+    =/  expiration  
+      (scan (trip (~(got by args) 'expiration')) dem)
+    [%create-gora-qr gor-id expiration]
   ==
 ::
 ++  build
   |=  $:  args=(list [k=@t v=@t])
           msgs=(unit [gud=? txt=@])
       ==
   ^-  reply:rudder
+
+  =/  arglist  (malt args)
+  =/  gor-id=@uv
+    :: TODO: @reviewers I replaced this with !! to avoid unwrapping
```

Comment:
Small comment here

Reply:
Just use `+got:by` instead. (Note that this is not a comment on whether crashing on non existence is safe in this context, it’s strictly a code style comment.)